packages/eventindexer/docs/docs.go

```diff
@@ -52,7 +52,7 @@ const docTemplate = `{
                 }
             }
         },
-        "/chartByTask": {
+        "/chart/chartByTask": {
```

Comment:
Do we also need to change this https://github.com/taikoxyz/taiko-mono/blob/main/packages/eventindexer/pkg/http/routes.go#L25 ?

Reply:
I change this because on https://eventindexer-swagger.taiko.xyz/#/ has not found result